addons/calendar/tests/test_event_recurrence.py

```diff
@@ -288,6 +288,24 @@ def test_ambiguous_full_day(self):
             (datetime(2020, 3, 30, 0, 0), datetime(2020, 3, 30, 23, 59)),
         ])
 
+    def test_videocall_recurrency(self):
+        self.event.videocall_location = self.event.discuss_videocall_location
+        self.event._apply_recurrence_values({
+            'interval': 1,
+            'rrule_type': 'weekly',
+            'mon': True,
+            'count': 2,
+        })
+
+        recurrent_events = self.event.recurrence_id.calendar_event_ids
+        detached_events = self.event.recurrence_id.calendar_event_ids - self.event
+        rec_events_videocall_locations = recurrent_events.mapped('videocall_location')
+        self.assertEqual(len(set(rec_events_videocall_locations)), 1, 'Recurrent events should have the same videocall location')
+        self.assertEqual(all([not(recurrent_event.videocall_channel_id.id) for recurrent_event in recurrent_events]), True, 'No channel should be set before the route is accessed')
```

Comment:
`all(not ...)` -> `not any(...)`.

Also the list comp is unnecessary, `not` is a unary operator (not a function), and accessing the `id` is optional, an empty recordset is also falsy.